Packs/AWS-Athena/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "AWS - Athena (Beta)",
+    "name": "AWS - Athena",
     "description": "Amazon Web Services Athena",
     "support": "xsoar",
-    "currentVersion": "1.0.8",
+    "currentVersion": "1.1.0",
```

Comment:
Should we make it a minor bump or a major bump? (2.0.0)

Reply:
Let's make it a major bump.